src/Vts/MonteCarlo/DataStructures/SimulationOptions.cs

```diff
@@ -36,10 +36,7 @@ public class SimulationOptions
             PhaseFunctionType = phaseFunctionType;
             Databases = databases;
             // check if databases list is null and if so make empty
-            if (Databases == null)
-            {
-                Databases = new List<DatabaseType>() { };
-            }
+            if (Databases == null) Databases = new List<DatabaseType> { };
             Seed = seed;

```

Comment:
I missed this one when we went through, same as before, add ( ) and remove { }

Reply:
Removed "{}" and added "()".

Reply:
Made the change and pushed.